docs/EVE-IMAGE-REPRODUCIBILITY.md

```diff
@@ -0,0 +1,286 @@
+# EVE Reproducibility
```

Comment:
It probably makes sense to put a link in BUILD.md to this document.

Reply:
Good idea. Adding in.